ping/ping.md

```diff
@@ -2,7 +2,7 @@
 
 | Lifecycle Stage | Maturity                 | Status | Latest Revision |
 | --------------- | ------------------------ | ------ | --------------- |
-| 3A              | Recommendation | Active | r0, 2022-11-04  |
+| 3A              | Recommendation | Active | r0, 2022-12-09  |
```

Comment:
None of the changes below 
```suggestion
| 3A              | Recommendation | Active | r0, 2022-11-04  |
```

As far as I can tell, none of the changes below change the semantics of the protocol, thus it is fine to leave this untouched.